site/content/zh-CN/docs/tasks/run/external_workloads/tektoncd.md

```diff
@@ -1,63 +1,68 @@
+此页面展示了在运行 [Tekton Pipeline](https://tekton.dev/docs/) 时，
+如何利用 Kueue 的调度和资源管理能力。
 
-This guide is for [batch users](/docs/tasks#batch-user) that have a basic understanding of Kueue. For more information, see [Kueue's overview](/docs/overview).
+本指南适用于对 Kueue 有基本了解的[批处理用户](/zh-CN/docs/tasks#batch-user)。
+欲了解更多，请参阅 [Kueue 概述](/zh-CN/docs/overview)。
 
-We demonstrate how to support scheduling Tekton Pipelines Tasks in Kueue based on the [Plain Pod](/docs/tasks/run_plain_pods) integration, where every Pod from a Pipeline is represented as a single independent Plain Pod.
+我们将演示如何基于 [Plain Pod](/zh-CN/docs/tasks/run_plain_pods) 集成，
+在 Kueue 中支持 Tekton Pipeline 任务的调度，其中来自 Pipeline 的每个 Pod 都表现为单个独立的 Plain Pod。
 
-## Before you begin
+## 开始之前
 
-1. Learn how to [install Kueue with a custom manager configuration](/docs/installation/#install-a-custom-configured-released-version).
+1. 学习如何[安装具有自定义管理器配置的 Kueue](/zh-CN/docs/installation/#install-a-custom-configured-released-version)。
 
-1. Follow the steps in [Run Plain Pods](docs/tasks/run/plain_pods/#before-you-begin) to learn how to enable and configure the `pod` integration.
+2. 按照[运行 Plain Pod](/zh-CN/docs/tasks/run/plain_pods/#before-you-begin)
+   中的步骤学习如何启用和配置 `pod` 集成。
 
-1. Check [Administrator cluster quotas](/docs/tasks/manage/administer_cluster_quotas/) for details on the initial Kueue step.
+3. 查看[管理员集群配额](/zh-CN/docs/tasks/manage/administer_cluster_quotas/)以获取初始 Kueue 步骤的详细信息。
 
-1. Your cluster has tekton pipelines [installed](https://tekton.dev/docs/installation/pipelines/).
+4. 确保你的集群已安装 Tekton Pipeline：[安装指南](https://tekton.dev/docs/installation/pipelines/)。
 
+## Tekton 背景   {#tekton-background}
 
-## Tekton Background
+Tekton 有 [Pipelines](https://tekton.dev/vault/pipelines-v0.59.x-lts/pipelines/)、
+[Tasks](https://tekton.dev/vault/pipelines-v0.59.x-lts/tasks/) 和
+[PipelineRun](https://tekton.dev/vault/pipelines-v0.59.x-lts/pipelineruns/) 的概念。
 
-Tekton has the concept of [Pipelines](https://tekton.dev/vault/pipelines-v0.59.x-lts/pipelines/), [Tasks](https://tekton.dev/vault/pipelines-v0.59.x-lts/tasks/) and [PipelineRun](https://tekton.dev/vault/pipelines-v0.59.x-lts/pipelineruns/).
+一个 pipeline 由多个 task 组成。Task 和 pipeline 必须在运行 pipeline 之前创建。
 
-A pipeline consists of tasks. Tasks and pipelines must be created before running a pipeline.
+一个 PipelineRun 运行 pipeline。
 
-A PipelineRun runs the pipeline.
+一个 TaskRun 运行单个 task。PipelineRun 将重用 TaskRun 来运行 pipeline 中的每个 task。
 
-A TaskRun runs a single task. PipelineRuns will reuse TaskRuns to run each task in a pipeline.
+### Tekton 定义   {#tekton-defintions}
 
-### Tekton Defintions
-
-As a simple example, we will define two tasks named sleep and hello:
+作为一个简单的例子，我们将定义两个名为 sleep 和 hello 的 task：
 
 {{< include "examples/pod-based-workloads/tekton-sleep-task.yaml" "yaml" >}}
 
 {{< include "examples/pod-based-workloads/tekton-hello-task.yaml" "yaml" >}}
 
-A pipeline composes these tasks.
+一个 pipeline 由这些 task 组成。
```

Comment:
```suggestion
一个 Pipeline 由这些 Task 组成。
```
If pipeline is kept to not translate, it's better to capitalize the first letter, to keep consistent in this context.